rpm/oidc-agent.spec

```diff
@@ -1,12 +1,11 @@
 Name: oidc-agent
-%define version %(cat VERSION)
-Version: %{version}
-Release: 1
+Version: 3.2.7
```

Comment:
Is this required to be hardcoded?
The version should still be read from the VERSION file.

Reply:
The problem is with "%(cat VERSION)" that assumes that the rpmbuild command is executed within the directory  where the VERSION file is, otherwise it fails with  "no such file or directory" which is the case when building on the OSG environment.

Reply:
I changed it so that the version is not hardcoded in the spec file but passed as a variable.
I also updated the makefile to pass the version.

If this approach is fine with you, we can merge it.